dev-docs/bidders/ssp_geniee.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: bidder
+title: Geniee SSP
+description: Geniee SSP Bidder Adapter
+biddercode: ssp_geniee
+userId: imuId
+media_types: banner
+safeframes_ok: false
+sidebarType: 1
+pbjs: true
+---
+### Note
+
+This is [Geniee](https://geniee.co.jp) Bidder Adapter for Prebid.js.
+(This is Geniee *SSP* Bidder Adapter. The another adapter named "Geniee Bid Adapter" is Geniee *DSP* Bidder Adapter.)
```

Comment:
Consider making this a 
```suggestion

{: .alert.alert-info :}
This is Geniee *SSP* Bidder Adapter. The another adapter named "Geniee Bid Adapter" is Geniee *DSP* Bidder Adapter.
```

You may also link the other DSP adapter

Reply:
I modified this.